morpheus/stages/input/kafka_source_stage.py

```diff
@@ -103,7 +105,11 @@ def __init__(self,
         if client_id is not None:
             self._consumer_params['client.id'] = client_id
 
-        self._topic = input_topic
+        if isinstance(input_topic, str):
+            input_topic = list(input_topic)
```

Comment:
Casting a string to list makes a list where each char is an element
```suggestion
            input_topic = [input_topic]
```